plugins/modules/inventory_workflow_manager.py

```diff
@@ -924,12 +929,12 @@ def get_existing_devices_in_ccc(self):
                         function='get_device_list',
                     )
                 offset = offset + 1
+                self.log("Received API response from 'get_device_list': {0}".format(str(response)), "DEBUG")
                 response = response.get("response")
                 if not response:
                     self.log("There are no device details received from 'get_device_list' API.", "INFO")
                     break
-
-                self.log("Received API response from 'get_device_list': {0}".format(str(response)), "DEBUG")
+                # self.log("ignore this Received API response from 'get_device_list': {0}".format(str(response)), "DEBUG")
```

Comment:
remove this line from the module if you are not printing the log.

Reply:
Updated the code

Reply:
Please don't comment the line, if not needed, remove it